src/pheval/config_parser.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+import yaml
+from serde import serde
+from serde.yaml import from_yaml
+
+
+@serde
+@dataclass
+class InputDirConfig:
```

Comment:
Can you add some documentation what exactly this class is for?

Reply:
added some documentation following how @souzadevinicius has structured his docstrings